ext/afform/core/managed/AfformType.mgd.php

```diff
@@ -1,66 +1,144 @@
   [
     'name' => 'AfformType:form',
     'entity' => 'OptionValue',
+    'cleanup' => 'always',
+    'update' => 'always',
     'params' => [
-      'option_group_id' => 'afform_type',
-      'name' => 'form',
-      'value' => 'form',
-      'label' => 'Submission Form',
-      'weight' => 0,
-      'icon' => 'fa-list-alt',
+      'version' => 4,
+      'values' => [
+        'option_group_id.name' => 'afform_type',
+        'name' => 'form',
+        'value' => 'form',
+        'label' => E::ts('Submission Form'),
+        'grouping' => NULL,
+        'filter' => 0,
+        'is_default' => FALSE,
+        'description' => NULL,
+        'is_optgroup' => FALSE,
+        'is_reserved' => FALSE,
+        'is_active' => TRUE,
+        'component_id' => NULL,
+        'domain_id' => NULL,
+        'visibility_id' => NULL,
+        'icon' => 'fa-list-alt',
+        'color' => NULL,
+      ],
+      'match' => ['option_group_id', 'name'],
     ],
   ],
   [
     'name' => 'AfformType:search',
     'entity' => 'OptionValue',
+    'cleanup' => 'always',
+    'update' => 'always',
     'params' => [
-      'option_group_id' => 'afform_type',
-      'name' => 'search',
-      'value' => 'search',
-      'label' => 'Search Form',
-      'weight' => 10,
-      'icon' => 'fa-search',
+      'version' => 4,
+      'values' => [
+        'option_group_id.name' => 'afform_type',
+        'name' => 'search',
+        'value' => 'search',
+        'label' => E::ts('Search Form'),
+        'grouping' => NULL,
+        'filter' => 0,
+        'is_default' => FALSE,
+        'description' => NULL,
+        'is_optgroup' => FALSE,
+        'is_reserved' => FALSE,
+        'is_active' => TRUE,
+        'component_id' => NULL,
+        'domain_id' => NULL,
+        'visibility_id' => NULL,
+        'icon' => 'fa-search',
+        'color' => NULL,
+      ],
+      'match' => ['option_group_id', 'name'],
     ],
   ],
   [
     'name' => 'AfformType:block',
     'entity' => 'OptionValue',
+    'cleanup' => 'always',
+    'update' => 'always',
     'params' => [
-      'option_group_id' => 'afform_type',
-      'name' => 'block',
-      'value' => 'block',
-      'label' => 'Field Block',
-      'weight' => 20,
-      'icon' => 'fa-th-large',
+      'version' => 4,
+      'values' => [
+        'option_group_id.name' => 'afform_type',
+        'name' => 'block',
+        'value' => 'block',
+        'label' => E::ts('Field Block'),
+        'grouping' => NULL,
+        'filter' => 0,
+        'is_default' => FALSE,
+        'description' => NULL,
+        'is_optgroup' => FALSE,
+        'is_reserved' => FALSE,
+        'is_active' => TRUE,
+        'component_id' => NULL,
+        'domain_id' => NULL,
+        'visibility_id' => NULL,
+        'icon' => 'fa-th-large',
+        'color' => NULL,
+      ],
+      'match' => ['option_group_id', 'name'],
     ],
   ],
   [
-    'name' => 'AfformType:system',
+    'name' => 'OptionGroup_afform_type_OptionValue_system',
```

Comment:
will this cause problems for anyone who already has a managed afform managed entity